### DISCUSSION TITLE
Enhancement: Fullscreen & Settings.

### DISCUSSION THREAD

**Discussion by SlavFries** [FunkinCrew/Funkin#1594] [Ideas]
## What is your suggestion, and why should it be implemented?
Hello.
As you know in the non-website version of the game has a really bad fullscreen,i mean when i play in "fullscreen" it has that White thing on the top,which is really annoying
ALSO
You could add to the game Settings.
So i would like if you could fix it and add settings to the game.
Have a great day.

**Comment by AlexGamingSW** [comment]
You can have a "real" fullscreen with the Alt+Tab command

**Comment by SlavFries** [comment]
> You can have a "real" fullscreen with the Alt+Tab command
The alt=tab changes the tab.

**Comment by BojayB** [comment]
If you press F11 it should fullscreen properly depending on what browser you use

**Comment by SlavFries** [comment]
> If you press F11 it should fullscreen properly depending on what browser you use

Im using it not on the browser and the f11 didnt work.

**Comment by BojayB** [comment]
how are you using it then? I have the game downloaded but it still runs in my browser.

**Comment by SlavFries** [comment]
> how are you using it then? I have the game downloaded but it still runs in my browser.

Im using the game downloaded.Like from the ZIP.

**Comment by BojayB** [comment]
So am I but I'm also on linux. Just got a mod but needed windows so I've got it on windows too. Try Alt+Enter

**Comment by Bloatatoe** [comment]
Use Alt-Enter on windows, it works perfectly.
